packages/io-services-cms-models/openapi/services-cms-schemas.yaml

```diff
@@ -1,3 +1,35 @@
+ServiceHistory:
+  type: object
+  description: Service basic data
+  properties:
+    continuationToken:
+      type: string
+      description: Continuation token for pagination
+    items:
+      type: array
+      items:
+        $ref: '#/ServiceHistoryItem'
+ServiceHistoryItem:
+  description: Service History model data
+  allOf:
+    - type: object
+      properties:
+        id:
+          type: string
+        status:
+          $ref: '#/ServiceHistoryStatusType'
+        version:
+          type: integer
+        version_date:
```

Comment:
**chore**: maybe you meant `last_update`?!
```suggestion
        last_update:
```

Reply:
be56741c58376b068d886a0950fb7ba5484142db